script.module.resolveurl/lib/resolveurl/plugins/streamembed.py

```diff
@@ -38,7 +38,7 @@ def get_media_url(self, host, media_id):
                 host, data.group(1), data.group(2)
             )
             html = self.net.http_GET(url, headers=headers).content
-            sources = re.findall(r'[A-Z]{10}=\d+x(?P<label>[\d]+)\s*(?P<url>[^\n]+)', html)
+            sources = re.findall(r'[A-Z]{10}=\d+x(?P<label>[\d]+).*?[\s*|h](?P<url>[^\n]+)', html)
```

Comment:
not sure if this is right `[\s*|h\` usually it is a list of characters so [\sh] makes sense